interaction.karma.conf.js

```diff
@@ -0,0 +1,36 @@
+const webpackConfig = require('./webpack.config');
+const path = require('path');
+
+delete webpackConfig.entry;
+
+module.exports = (config) => {
+  config.set({
```

Comment:
This is basically the same as the regular config, so I'd export a function that creates a default config, which a config file further down can import and modify as needed.

Reply:
Created a file within the test tools directory to contain the new function.